cabot/metricsapp/forms/grafana.py

```diff
@@ -61,8 +61,11 @@ def __init__(self, *args, **kwargs):
         )
 
         if default_panel_id is not None:
-            default_panel = next(panel for panel in panels if panel[0]['panel_id'] == default_panel_id)
-            self.fields['panel'].initial = default_panel[0]
+            for panel in panels:
+                panel_data = panel[0]
+                if panel_data['panel_id'] == default_panel_id:
+                    self.fields['panel'].initial = panel_data
+                    break
```

Comment:
Does this whole thing require there to be an initial panel? Does it makes sense to put before the for loop
```
self.fields['panel'].initial = panels[0][0]
```

Reply:
na you don't need a initial panel, it'll just default to the first one if you don't set initial